### PR TITLE
ci: fix appveyor

### LINKED ISSUE #1062: AppVeyor broken???
## Type of Issue

```
[x] Bug Report
[ ] Feature Request
```

## Description

```bash
Running Install scripts
Install-Product node $env:nodejs_version
7z : The term '7z' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
At C:\Program Files\AppVeyor\BuildAgent\Modules\avvm\avvm.psm1:100 char:13
+             7z x $zipPath -y -o"$path" | Out-Null
+             ~~
    + CategoryInfo          : ObjectNotFound: (7z:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
 
Uninstalling node 8.11.3 (x86)...
7z : The term '7z' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
At C:\Program Files\AppVeyor\BuildAgent\Modules\avvm\avvm.psm1:100 char:13
+             7z x $zipPath -y -o"$path" | Out-Null
+             ~~
    + CategoryInfo          : ObjectNotFound: (7z:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
 
Installing node 8.11.4 (x86)...
node --version
'node' is not recognized as an internal or external command,
operable program or batch file.
Command exited with code 1
```

#### How To Reproduce

See logs from most recent CI builds.

#### Expected Behaviour

green lights 🍏 

#### Version Information

n/a

### PR DESCRIPTION
**Pull request by alan-agius4**
Closes #1062

### CODE REVIEW

File: .appveyor.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ environment:
   global:
     # Set %PATH% to avoid msys/cygwin DLL conflicts which causes:
     # 'fatal error - cygheap base mismatch detected'
-    PATH: C:\msys64\usr\bin;C:\WINDOWS\system32;C:\WINDOWS;C:\Program Files (x86)\Yarn\bin;C:\Program Files (x86)\nodejs
+    PATH: C:\msys64\usr\bin;C:\WINDOWS\system32;C:\Program Files (x86)\7-Zip\7z.exe;C:\WINDOWS;C:\Program Files (x86)\Yarn\bin;C:\Program Files (x86)\nodejs
```
Comment (JoostK) [inline .appveyor.yml]: I don't think `7z.exe` should actually be included in `PATH`?

**Comment by dherges** [comment]
Shall I merge?

**Comment by alan-agius4** [comment]
Can you enable app veyor and try to force a build on this Pr?

**Comment by dherges** [comment]
AppVeyor is running again on master branch

**Comment by dherges** [comment]
Think someone has to push to this branch to trigger ApPVeyor for the PR again?!?

**Comment by alan-agius4** [comment]
Done

**Comment by alan-agius4** [comment]
Superseeded by https://github.com/ng-packagr/ng-packagr/pull/1085

**Pull request closed without merging**